### DISCUSSION TITLE
General Minor Suggestions

### DISCUSSION THREAD

**Discussion by secretmeister** [secretmeister/NeutronHeaven#6] [General]
Leave any minor suggestions here that don't deserve an entire issue. I will respond to all I see. Please do put in suggestions, I really need the feedback.

**Comment by sbossarte** [comment]
It's been a solid experience so far, lots of having to figure things out, which I like in a pack like this.

I am a bit stumped on how to get rain/thunderstorms for the water and soul glass, as it seems by default the doWeatherCycle rule is set to false and there's no items I can see that would otherwise change the weather.

**Comment by secretmeister** [comment]
That is one thing I should explain better, look in slimy synergy for the answer.

**Comment by sbossarte** [comment]
I see where getting the trident updates the weather, but I'm not sure how to get to that point without having the weather there in the first place, since the only way to get prismarine is with bulk haunting, which would require soul glass to seed that process. Seems like a loop, but I figured I was missing something, and wanted to ask.

**Comment by secretmeister** [comment]
AH, you would be correct. That is something that will be adressed very shortly.

**Comment by LarcekGiglamesh** [comment]
i'd still suggest to use discord instead of github. to start with, not every player has an account on curseforge.com and even less have one for github.com, but plenty of will likely own an account by now for discord. 

Minor Suggestion (as of v.1.5.2)
- Checkmark Quest to get the first Cobblestone with a Hint towards the Emergency Items
- Better Description on how to Summon Entities, or adding JEI References when looking up usages of Bucket of Blood
 -- i'd still prefer getting spawn eggs, but that's just personal preference
- Replace Lychee Tweaker with something else if possible, it's not as readable as other means
- Recipe for Overgrown Seed would be nice
- Better Option to make Lava lategame
- Better Morph Ratio for Sandstone to End Stone
- Remove the Weather Changes upon summoning Entities (Slimes cause Rain?)
- Better Description for End Painting Quest, regarding the usage of the Painting
- Reward an Overworld Painting once you completed the End Painting Quest

**Reply by secretmeister** [reply]
I... don't quite know what you mean by weather changes on summoning entities, if those do exist it is not on purpose, nor in my scripts.

I have changes in place to solve the first and second to last

If you have a suggestion for an alternative to LycheeTweaker I would consider it, but I am not going to write it out in json, that would cause way more problems than it would solve. The other tweaks I will work on now, including a quest for the overworld painting so you aren't stuck. 

As for the conversation platform, I am still in school and don't have the time or resources to setup an official and well moderated discord, and the combination of these two options should serve most people. I also do have a channel on the general Minecraft Pack Development discord.

**Comment by PashaFeed** [comment]
Im not so good at english, so im not sure if this is the right place to speak bout this, but i am traly stuck at one quest and i realy need help with it, there is a questfor getting nether bones, and the are only 2 ways i found to get them, first one is slaying wither skeletons, who can only be spawned using those bones, and the second way is using mixing from create, which requires not only blaze burner, which quest needs nether bones quest yo be done firstly, bit also requires wood coal which requirrs wood which is in the next chapter just as a blaze burner, so how do i make this quest, without breaking the chapter thing? Also without shortcuts i guess
